### ISSUE TITLE
Disabled instrumentation tests are reported as 'passing' in HTML test summaries

### ISSUE THREAD

**Issue by ktkopone** [mannodermaus/android-junit5#193]
Disabled instrumented tests (apart from disabled parameterized tests; see https://github.com/gradle/gradle/issues/10526) appear to be reported correctly as 'skipped' in console output, e.g.:
```
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: class=StupidConnectedTest
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: current=8
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: numtests=6
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: stream=
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: test=stupidConnectedDisabledFailTest()
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS_CODE: 1
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: class=StupidConnectedTest
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: current=8
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: numtests=6
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: stream=
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS: test=stupidConnectedDisabledFailTest()

StupidConnectedTest > stupidConnectedDisabledFailTest()[Nexus 5X - 8.1.0] SKIPPED 
04:37:20 V/InstrumentationResultParser: INSTRUMENTATION_STATUS_CODE: -3
```

and also reported correctly as skipped in xml reports. However, in the HTML test report files, disabled instrumented tests are reported as passing:

<img width="552" alt="Screen Shot 2019-11-12 at 4 34 15 PM" src="https://user-images.githubusercontent.com/26582629/68712713-4d504580-056a-11ea-8a53-1bcc0f41919e.png">

For comparison, disabled unit tests are reported as skipped appropriately in their summary:

<img width="913" alt="Screen Shot 2019-11-12 at 4 32 07 PM" src="https://user-images.githubusercontent.com/26582629/68712595-0c583100-056a-11ea-909e-165a48c6e06a.png">

May be a gradle-side issue, I'm not sure at what point the HTML report generation happens.

Env: JUnit 5.5.1, mannodermaus plugin 1.5.1.0, android-test-core:1.0.0, android-test-runner:1.1.0 (still an issue as of 1.2.0 snapshot).

**Comment by mannodermaus** [comment]
I believe this also happens with JUnit 4-style instrumentation tests. Might be an Android issue. The execution is skipped as expected for both JU4 and JU5, but the report shows both as `passed`:

```
// BaselineJUnit4Test.kt:
@Ignore
@Test
fun ignoredTest() {
}

// JavaInstrumentationTests.kt (JUnit 5):
@Disabled
@Test
void testDisabled() {
}
```

In the report:

|||
|---|---|
|<img width="657" alt="Screen Shot 2019-11-15 at 21 37 29" src="https://user-images.githubusercontent.com/3897705/68974113-3ae73d80-07f0-11ea-9ff5-e49d7e9146bb.png">|<img width="738" alt="Screen Shot 2019-11-15 at 21 37 35" src="https://user-images.githubusercontent.com/3897705/68974114-3ae73d80-07f0-11ea-85c1-2c8f6ee9be00.png">|

**Comment by mannodermaus** [comment]
After another investigation tonight, I was able to pinpoint this to a bug in the Android Gradle Plugin. I filed [an issue on the Google issue tracker](https://issuetracker.google.com/issues/195108337). As this problem affects JUnit 4 and 5 in the same manner, I'm closing this ticket in favor of the Google issue.

**Comment by mannodermaus** [comment]
@ktkopone FYI, this has been fixed in Android Gradle Plugin 7.1.0 Alpha 9, which released today. From now on, JUnit 5 tests (and JUnit 4 too) should be reported as ignored correctly.
